docs/en/prerequisites.rst

```diff
@@ -1,6 +1,49 @@
 Prerequisites
 ===============================
-The minimum requirements for running the ESP-IDF Installation Manager are below.
+Below are the minimum requirements for running the ESP-IDF Installation Manager.
+
+Windows
+----------------
+
+- git
+- cmake
+- ninja
+- python with pip capable of creating virtual environment and doing SSL requests
+
+.. note::
+
+    If any of these prerequisites are missing, the installer will prompt you to install them. If you agree, the installer will automatically install and configure everything required to run ESP-IDF.
+
+MacOs
+----------------
+
+- dfu-util
+- cmake
+- ninja
+- python with pip capable of creating virtual environment and doing SSL requests
```

Comment:
Mention only the prerequisites that are not installed by eim but are needed for IDF.

Reply:
we do not install any prerequisities on posix systems so all this are needed for some version of IDF